repository/kafka/docs/latest/kudo-kafka-runbook.md

```diff
@@ -0,0 +1,128 @@
+# KUDO Kafka Runbook
```

Comment:
add `kudo-kafka-runbook.md` to the `v1.0` directory also

Reply:
Done!